index.html

```diff
@@ -5,8 +5,76 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel='stylesheet' href='styles/index.css'/>
 </head>
+
+
 <body>
+
     
+
+    <section class='wholeThing'>
```

Comment:
I would choose class names that would be easy for other developers to understand. So you could use "page-container" or something similar.